bodhi/tests/server/models/__init__.py

```diff
@@ -32,9 +32,10 @@ class ModelTest(object):
     klass = None
     attrs = {}
 
-    def setup(self):
+    def setUp(self):
         engine = create_engine('sqlite://')
-        Session = scoped_session(sessionmaker(extension=ZopeTransactionExtension()))
+        Session = scoped_session(
+            sessionmaker(extension=ZopeTransactionExtension(keep_session=True)))
         Session.configure(bind=engine)
```

Comment:
I attempted to use inheritance here but these tests do not work properly with that superclass. I plan to return to these later to debug that, but I'd rather keep this PR focused on what is needed to fix and test the e-mail issue.